src/Components/PaperEditor/TranscriptsContainer/Paragraphs/index.js

```diff
@@ -1,169 +0,0 @@
-import PropTypes from 'prop-types';
```

Comment:
This file was broken down into several different functions etc in `TranscriptContainerTab` as part of refactoring.